repos/system_upgrade/el7toel8/actors/localreposinhibit/actor.py

```diff
@@ -0,0 +1,86 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import TargetRepositories, UsedTargetRepositories
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+
+
+class LocalReposInhibit(Actor):
+    """Inhibits the upgrade if local repositories were found.
```

Comment:
```suggestion
    """
    Inhibits the upgrade if local repositories were found.
```

it's style used in majority code in leapp project it looks really much better. but not blocker from me. if you change it, change it everywhere in this PR, it's on more places.

Reply:
@pirat89  This style is accepted in https://www.python.org/dev/peps/pep-0257/#id17 . I will fix it in all other places . Thx